### DISCUSSION TITLE
Run Dependabot on private runner

### DISCUSSION THREAD

**Discussion by ericvercherat** [community/community#39728] [Enterprise]
### Dependabot

What is the configuration required to run Dependabot on private runner ? 

### Body

Hi !

How to configure dependabot to run on self hosted runner ? Seems like things evolved since 3.3 github entreprise version.

We followed this documentation : https://docs.github.com/en/enterprise-server@3.4/admin/github-actions/enabling-github-actions-for-github-enterprise-server/managing-self-hosted-runners-for-dependabot-updates

- Private runner configured with succes (docker and git)
- Label "dependabot" added

We dont find any configuration option (https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file) to use custom runner just like in other actions. 

We use this configuration in our CI/Action tu run php tests **with sucess** : 

`jobs:
    # Runs backend unit tests with Phpunit
    phpunit:
        name: PHP TEST
        runs-on: ['private-runner', 'php']`

Linked discussion : https://github.com/orgs/community/discussions/33826

Thanks,
Eric

**Comment by guillaumepeano** [comment]
Hi, got the same issue here. 

We've followed same documentation, runners are right configured (with right label), but automatic or manual search for updates not firing analysis on our runner.

Any updates regarding this issue/misconfiguration ?

Thanks,

Guillaume
